test/test_average_waiting_time.c

```diff
@@ -0,0 +1,53 @@
+#include <stdio.h>
+#include "../include/globals.h"
+#include "../include/average_waiting_time.h"
+
+/**
+ * @file test_average_waiting_time.c
+ *
+ * @author Ritika Arora
+ *
+ * @brief tests the  average_waiting_time function if working fine or not
+ *
+ * @detail The function is to test average_waiting_time function for given inputs, the function
+ * returns right values or not. The average_waiting_time function is called with different inputs
+ * and its output is stored in one local variable. The max_time local variable should give the
+ * maximum of all the inputs given.
+ * The function checks if the max_time stores maximum of the inputs given to the max_waiting_time
+ * function , then the test passes and prints the result on console as well as writes on a file.
+ *
+ * @param[in] No input is given to the function
+ *
+ * @return void The function returns nothing.
+*/
+
+
+void test_average_waiting_time (){
+
+	FILE* test_results = fopen("R:\\Ritz\\elevator_simulator\\test\\results\\results_average_waiting_time.txt", "w+");
```

Comment:
you can put the path as fopen("test\\results\\results_average_waiting_time.txt", "w+") or
fopen("test/results/results_average_waiting_time.txt", "w+")

Reply:
Corrected the path in the file